tests/versions/index.js

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env node
+/* eslint-disable no-process-exit, no-console */
+
+// This file does not use any dependencies, so that it can be ran before installing
+
+// It checks whether the versions of packages that should be versioned synchronously
+// are actually in sync. We need this as long as Lerna doesn't have a mixed mode.
+// see: https://github.com/lerna/lerna/issues/1159
```

Comment:
Here's the script to check version numbers are sync @sarahdayan

Reply:
Beautiful!